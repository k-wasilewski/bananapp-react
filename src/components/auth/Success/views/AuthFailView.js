import React, {Component} from "react";
import {Link} from "react-router-dom";

class AuthFailView extends Component {
    render() {
        return (
            <div className="App">
                <header className="App-header">
                    <h3> Login failed, try again</h3>
                    <Link to="/">
                        <button variant="outlined">
                            Back
                        </button>
                    </Link>
                </header>
            </div>
        )
    }
}
export default AuthFailView;