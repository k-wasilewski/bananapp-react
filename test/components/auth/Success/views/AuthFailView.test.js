import React from 'react';
import renderer from 'react-test-renderer';
import store from "../../../../../src/redux/store";
import {BrowserRouter} from "react-router-dom";
import AuthFailView from "../../../../../src/components/auth/Success/views/AuthFailView";
import {Provider} from "react-redux";

describe("AuthFailView rendering specification", () => {
    it('AuthFailView is rendered', () => {
        const component = renderer.create(
            <Provider store={store}>
                <BrowserRouter>
                    <AuthFailView />
                </BrowserRouter>
            </Provider>
        );
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});