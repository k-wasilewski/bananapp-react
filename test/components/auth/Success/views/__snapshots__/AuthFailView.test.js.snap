// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthFailView rendering specification AuthFailView is rendered 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <h3>
       Login failed, try again
    </h3>
    <a
      href="/"
      onClick={[Function]}
    >
      <button
        variant="outlined"
      >
        Back
      </button>
    </a>
  </header>
</div>
`;
